### DISCUSSION TITLE
2023-11-09

### DISCUSSION THREAD

**Discussion by ngsy** [10-rolling/rollingPolling#46] [Daily_Meeting]
### ✅ 어제 해낸 일

- [x] 메세지 보내기 구현 
- [x] 카드컴포넌트 구현
- [x] Post 페이지 구현
- [x] Header 컴포넌트 구현
- [x] react hook -> zustand hook으로 변경
- [x] 라이센스 등록
- [x] modal 구현
- [x] main page 구현

### 📝 회의내용

- TMI (아이스브레이킹)
마이멜로디 케이크 주문했습니다. 수영
오늘 끝나면 쇼핑가야해 방해하지마. 슬희
오늘의 DJ
jvke - this is autumn feels like  윤혁  <a href="https://www.youtube.com/watch?v=fk3quqey7I4"> 가사 보면서 들으면 좋아요 </a>
오빠네 강아지가 놀러왔다. 포시가 삐져서 각방썼다. 수

- 현재 진행상황 공유 
수: 헤더 컴포넌트 다 만들었고  물론! 반응형은 아직 안했다.
윤혁 :메세지 컴포넌트 서버에 메세지 보내는것까지 만들었다.
슬희: API , 메인페이지 , 수정했고 , 모달 , 카드컴포넌트,  카드리스트 컴포넌트(마크업만하였음) 
수영: 포스트페이지 구현 서버로 보내는건 아직 

- 구현한 것 발표
 다들 고생하셨습니다.

### 💪🏻 오늘 할일

- 수 님 

- 슬희 님

- 윤혁 님

- 수영 님

**Comment by hanseulhee** [comment]
반성 시간 ㅋㅋㅋㅋㅋㅋㅋ 몬일이죠

**Reply by ngsy** [reply]
우당탕탕 합치는 시간이 될꺼같아서요ㅋㅋㅋㅋ😔

**Comment by oauch** [comment]
드디어 합체 작업(?) ⚙️
